cmd/go.mod

```diff
@@ -17,7 +17,7 @@ require (
 	golang.org/x/sys v0.16.0
 	google.golang.org/grpc v1.58.3
 	k8s.io/cri-api v0.28.2
-	oras.land/oras-go/v2 v2.3.1
+	oras.land/oras-go/v2 v2.4.0
```

Comment:
Release notes https://github.com/oras-project/oras-go/releases/tag/v2.4.0

A few items to note:
1. Adds deletion and garbage collection in oci package.
* Note: this will enable a path to resolve the orphaned zTOCs issue for the SOCI content store.
2. Improves performance for copy graph operations by adding support for mounting on copy.
* Note: copy graph occurs on SOCI index push. Currently unclear if this is a realistic improvement for SOCI use case.
3. The bug fix to prevent a panic on index or manifest list with optional platform is not an issue for SOCI.
* Note: SOCI with throw an error if platform is empty. See https://github.com/awslabs/soci-snapshotter/blob/db7df3ab2a840bc927417a84448263885b3e21ff/soci/soci_index.go#L505-L507